realm/realm-library/src/androidTest/java/io/realm/RealmTests.java

```diff
@@ -781,6 +781,7 @@ public void realmConfiguration_fileName() {
     }
 
     @Test
+    @Ignore
```

Comment:
Do we really want to disable it? Can we ignore it when using a custom core only?